src/Examples/Advanced/CustomProperties/Dictionary_Txy_diagrams.ipynb

```diff
@@ -0,0 +1,339 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Create a dictionary and use Txy diagrams"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Introduction\n",
+    "\n",
+    "This Jupyter Notebook creates a property package for the generic property package and utilized the Txy diagrams for the generic properties. A dictionary with the neccesary parameters.\n",
```

Comment:
The second sentence here seems incomplete.

Also, I think a little more description of what this example shows and what the user will achieve by the end would be useful.